lib/workers/repository/dependency-dashboard.ts

```diff
@@ -117,7 +117,7 @@ export async function ensureDependencyDashboard(
       )
     )
   ) {
-    if (GlobalConfig.get('dryRun')) {
+    if (GlobalConfig.get('dryRun') === 'full') {
```

Comment:
I think if we got this far then you no longer need to check for the value of `dryRun` - just the presence of it. So all these `=== 'full'` additions could be removed.